FAQ.md

```diff
@@ -23,7 +23,7 @@ this.setState({ editorState: EditorState.createEmpty() })
 
 ## Why are mentions broken after using `convertFromRaw` and throwing an error?
 
-We design the API to accept an Immutable Map for a Mention. After saving your datastructure to the erver and using `convertFromRaw` the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: We might fix this with v2.0.0 f the mentions plugin.
+We design the API to accept an Immutable Map for a Mention. After saving your data structure to the server and using `convertFromRaw`, the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: we might fix this with v2.0.0 of the mentions plugin.
```

Comment:
great conversion! very much what I was thinking about as well.

In general we could allow both, but I wonder if it creates more confusion than it actually help. I like it when there is only one way of doing things. Then docs and examples out in the wild are consistent. Initially I wanted to go with Immutable all the way, but due the nature of `convertFromRaw` plain JS objects make more sense.

Right now I'm thinking: Let's do both for now, convert everything to plain object examples/docs and eventually drop Immtuable (if not other API changes happened until then in Immutable or `convertFromRaw`.